### DISCUSSION TITLE
Importing JSON or ndjson in duckdb: is there a guide to do it?

### DISCUSSION THREAD

**Discussion by aborruso** [duckdb/duckdb#4264] [Q&A] [answered]
Thank you

**Comment by Mause** [comment]
Have you looked at the docs for the json extension? 

https://duckdb.org/docs/extensions/json

**Reply by aborruso** [reply]
> Have you looked at the docs for the json extension?

Thank you @Mause. 
How do i import a file? I had seen that page, but I don't see a reference to importing a file.

**Reply by Mause** [reply]
Which duckdb interface are you using?

**Reply by aborruso** [reply]
You are right, sorry.

I'm using the CLI.

Thank you

**Reply by Alex-Monahan** [reply]
Check out these options!
#3774

**Reply by Mause** [reply]
> Check out these options! #3774

Looks like the `read_json_objects` and `read_ndjson_objects` macros are missing from the json extension docs?

**Reply by aborruso** [reply]
> Looks like the `read_json_objects` and `read_ndjson_objects` macros are missing from the json extension docs?

I felt foolish not to see these functions. 
Now I have confirmation, that these are not in the guide.

Thank you all, very useful answers

**Comment by aborruso** [comment, marked as answer]
Then to import via CLI a JSON (thank you @Alex-Monahan ), I can use `read_json_objects` and `read_ndjson_objects` macros (#3774)
